### DISCUSSION TITLE
[modal] how to hide the backdrop while be able to click outside to close the modal on bootstrap 5

### DISCUSSION THREAD

**Discussion by handhikadj** [twbs/bootstrap#37227] [General]
`data-bs-backdrop="false"` hides it but cannot click outside to close the modal

**Comment by ascensus-mdb** [comment]
Something like this?
https://mdbootstrap.com/docs/standard/components/modal/#section-non-invasive-modal

**Reply by handhikadj** [reply]
I think I didn't make this post clear. updated it
